src/test/java/io/jenkins/plugins/report/jtreg/main/comparator/PrintTableTest.java

```diff
@@ -1,33 +1,37 @@
 package io.jenkins.plugins.report.jtreg.main.comparator;
 
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
+import io.jenkins.plugins.report.jtreg.main.comparator.formatters.Formatters;
```

Comment:
Actually.. maybe you can reuse formatters form DIFF? io.jenkins.plugins.report.jtreg.main.diff.formatters?

Reply:
Tehy are VERY strghtforward.